### DISCUSSION TITLE
Disable dbms output

### DISCUSSION THREAD

**Discussion by hanulful** [dbeaver/dbeaver#13172] [Q&A]
- I'm trying to connect to the **Oracle** db
- I only have **select-only permission**

I want to connect to the database without DBMS_OUTPUT.enable option
Is there any option something like the DBeaver's setting or ojdbc connection properties when connecting to db?

p.s.
The connection fails maybe because of the DBeaver uses...
"
BEGIN
  DBMS_OUTPUT.Enable(1000000);
END;
"

thanks

**Comment by LonwoLonwo** [comment]
Hello @hanulful 
Please try to disable "Enable DBMS Output" setting in Preferences->Connections->Drivers->Oracle

![2021-07-15 10_18_39-Window](https://user-images.githubusercontent.com/45152336/125746114-710f3ab2-ad2f-4a90-9a80-32434c80468e.png)

**Comment by hanulful** [comment]
At that time I only had READ-ONLY permission and some PL/SQL control.
I set a the read-only options, 
but the problem happened because the DBeaver committed even using a SELECT query.

The _"SMART COMMIT MODE_ *" solved this..
https://dbeaver.com/docs/wiki/Auto-and-Manual-Commit-Modes/
_*In this mode, the transaction will only be started when user start to modify user's data._


and thank you @LonwoLonwo
